docs/assets-update.md

```diff
@@ -0,0 +1,66 @@
+# Clear Containers assets updates 
+
+## Clear Containers Kernel
+
+[Clear Containers kernel](https://github.com/clearcontainers/linux). It is a
+Linux kernel based in the latest LTS version. A LTS version is used to  reduce
+the number of updates we'd have to roll out and use a more 'stable' and tested
+kernel. 
+
+### Clear Containers Kernel update
+
+Clear Containers release is done weekly. Each  time a new kernel version is out
+should be updated updated in 
+[Clear Containers packaging repository](https://github.com/clearcontainers/packaging)
+and [Clear Containers Linux kernel](https://github.com/clearcontainers/linux)
+repository. The the last version updated in this repositories will be used for
+the release. In case of any critical update is done in the kernel a new Clear
+Containers Release will need to be done immediately otherwise the next Clear
+Containers release will include the changes done in the week.
+
+### Owner 
+
+Miguel Bernal (@miguelinux) is primarily owner for Linux Kernel update in
+https://github.com/clearcontainers/linux and Clear Linux\*
+
+### Clear Containers Image.
+
+The Clear Containers image guest OS system is generated using Clear Linux packages. This image 
+Contain a minimal system for Clear Containers. The image is generated twice a day as
+part of the Clear Linux Release Cycle. 
+
+### Update Clear Containers image.
+
+The Clear Containers image should be updated only when critical updates were
+done to the packages used by Clear Containers Guess OS:
+
+- Systemd
+- Clear Containers Agent
+- iptables
+- core-utils
+
+This can must be verified each release using the script
+https://github.com/clearcontainers/packaging/blob/master/scripts/get-image-changes.sh
+
+The image must be updated in
+[Clear Containers packaging repository](https://github.com/clearcontainers/packaging)
+and defined in https://github.com/clearcontainers/runtime/blob/master/versions.txt
+in the [Clear Containers Release](https://github.com/clearcontainers/runtime/wiki/Release-Process).
+
+### Owner
+
+Carlos Venegas (@jcvenegas) current Clear Containers Release owner will verify
+this each release.
+
+## Clear Containers Continues integration.
+
+Official Clear Containers Packages are hosted in OBS build system. In order to
+avoid availability issues the kernel and image for Clear Containers Continues 
+integration system used is downloaded from Clear Linux\* packages:
+
+```
+https://download.clearlinux.org/releases/${CLEAR_VERSION}/clear/x86_64/os/Packages/linux-container-${KERNEL_VERSION}.rpm 
+```
+
+The packages from Clear Linux provide the same kernel and image used in other
+Linux Distributions. And can be  safely used without any compatibility issue.
```

Comment:
I'd change these two sentences to something like, "The Clear Containers assets packages provide identica kernel and images for the various Linux Distributions." I'm not sure we need the 2nd sentence tbh.